src/lib/Behat/BusinessContext/AdministrationContext.php

```diff
@@ -278,4 +278,24 @@ public function listIsEmpty(string $listName, string $itemType, string $itemName
         $pageObject = PageObjectFactory::createPage($this->utilityContext, $itemType, $itemName);
         $pageObject->verifyListIsEmpty($listName);
     }
+
+    public function findElementOnTheList(Page $pageElement, string $listElementName): bool
```

Comment:
Either this function should return Element or be named `isElementOnTheList`

Reply:
I called it `findElementOnTheList` because it searches for the element through the pages and `isElementOnTheList` is already taken by one of the step functions. Maybe I will just change it to `is` and step functions to `verify` as we call it in other places.